src/components/InputTodo.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+
+class InputTodo extends Component {
+  state = {
+    title: ''
+  }
+
+  onChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    })
+  }
+
+  handleSubmit = e => {
+    e.preventDefault()
+    this.props.addTodoProps(this.state.title)
+  }
+
+  render () {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <input
+          type='text'
+          placeholder='Add todo...'
+          name='title'
+          value={this.state.title}
+          onChange={this.onChange}
+          addTodoProps={this.addTodoItem}
```

Comment:
Kindly, make sure to add props that actually exist. That way your code will look much cleaner & easier to understand.